src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -550,7 +554,7 @@ export class MgtPersonCard extends MgtTemplatedComponent {
       this.personImage = this.getImage();
     } else if (this.personQuery) {
       // Use the personQuery to find our person.
-      const people = await findPerson(graph, this.personQuery);
+      const people = await findPeople(graph, this.personQuery, 1);
```

Comment:
Use the `PersonType` enum rather than simply the number.

Reply:
The number here is for the number of people to return